src/shared-components/dialogModal/style.ts

```diff
@@ -46,7 +46,7 @@ export const ModalContainer = styled.div<{
   background: ${({ backgroundColor }) => backgroundColor};
   padding: ${SPACER.x4large} ${SPACER.large} ${SPACER.xlarge};
   overflow-y: auto;
-  max-height: 700px;
+  max-height: 100%;
```

Comment:
This fixes the `iPhone5` resolution scenario, but then creates a sub-optimal experience on Desktop, taking up the whole viewport, which is not the intent of the `DialogModal`:

<img width="491" alt="Screen Shot 2021-02-12 at 3 51 20 PM" src="https://user-images.githubusercontent.com/13544620/107826275-42c2c400-6d4a-11eb-9003-f2cce5ad8ac3.png">

You may be able to wrap the styling in a media query in order to have it only apply to low-resolution devices. However, this seems to me like a violation of the distinction we try and maintain between `DialogModal` and `ImmersiveModal`: "Dialog modals shouldn't contain large content and should not scroll unless screen size dictates it. To display large amounts of content, use Immersive modal instead.". I think this example qualifies as large amounts of content. 

I would try and see if you can wrap `<UrgentTriageModal />` in an `ImmersiveModal` instead of a `DialogModal`. You will likely have to make a few changes if you proceed down this route (e.g. making sure the `Report a Medical Emergency` title persists in the transition as you scroll down). I would consult with stakeholders depending on the route you take--I can follow up later if you'd like more guidance but I have a meeting right now and wanted to get the comment to you 😬

Reply:
I'll take a look at the `ImmersiveModal` and may reach out on Monday with more questions!